### PR TITLE
Mob group/pool QC for beastmen in past vs present zones

### LINKED ISSUE #838: Tiny Turtles in Qulun
**_I have:_**

- [x] searched existing issues (http://project-topaz.com/issues/) to see if the issue has already been opened
- [x] checked the commit log to see if the issue has been resolved since my server was last updated

**_Additional Information_** (Steps to reproduce/Expected behavior) **:** 

Ruby, adaman, and ancient (I think) quadavs in Qulun dome suffer from tinyism.  This is an issue with their flag settings, not model id, as discussed on Discord.
image

### PR DESCRIPTION
**Pull request by wrenffxi**
**_I affirm:_**
- [x] that I agree to Project Topaz's [Limited Contributor License Agreement](http://project-topaz.com/blob/release/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I've _tested my code_ since the last commit in the PR, and will test after any later commits

Set pools based on Nyu's retail captures for past vs present zones.
* Quadavs [Fixes #838]
* Yagudos
* Orcs

### CODE REVIEW

File: sql/mob_pools.sql [hunk 1]
```diff
@@ -760,7 +760,7 @@ INSERT INTO `mob_pools` VALUES (699,'Chasmic_Hornet','Chasmic_Hornet',48,0x00001
 INSERT INTO `mob_pools` VALUES (700,'Chasm_Coeurl','Chasm_Coeurl',71,0x00006F0100000000000000000000000000000000,1,1,2,240,100,0,1,0,0,0,0,0,0,0,0,0,0,1,0,71);
 INSERT INTO `mob_pools` VALUES (701,'Chasm_Gnat','Chasm_Gnat',131,0x0000DD0700000000000000000000000000000000,6,6,3,240,100,0,1,0,0,0,0,0,0,0,0,0,0,1,0,131);
 INSERT INTO `mob_pools` VALUES (702,'Chasm_Lizard','Chasm_Lizard',174,0x0000480100000000000000000000000000000000,1,1,7,240,100,0,0,0,1,0,0,0,565,131,0,0,0,1,0,174);
-INSERT INTO `mob_pools` VALUES (703,'Chatoyant_Quadav','Chatoyant_Quadav',202,0x00001E0400000000000000000000000000000000,3,3,6,265,100,0,1,0,1,0,0,0,1,1157,0,0,1,1,0,202);
+INSERT INTO `mob_pools` VALUES (703,'Chatoyant_Quadav_BS','Chatoyant_Quadav_BS',202,0x00001E0400000000000000000000000000000000,3,3,6,265,100,0,1,0,1,0,0,0,1,131,0,0,1,1,0,202);
```
Comment (TeoTwawki) [inline sql/mob_pools.sql:763]: "BS"?
sorry the abbreviations/initialing pattern isn't hitting me.

Comment (wrenffxi) [reply sql/mob_pools.sql:763]: _BS = Beadeaux [S].  Some of the mobs needed multiple pools even within past zones, like the ones in Beadeaux [S] and the ones in Castle Zvahl Keep [S] have different flags and/or models.

Comment (TeoTwawki) [reply sql/mob_pools.sql:763]: got it, thanks.

**Comment by wrenffxi** [comment]
Added yagudos and orcs to this PR.

**Review by cocosolos — APPROVED** [review]

**Review by ibm2431 — APPROVED** [review]

**Pull request merged**